tests/lib/mem_blocks/src/main.c

```diff
@@ -205,6 +205,182 @@ static void test_mem_block_multi_alloc_free_alt_buf(void)
 	alloc_free(&mem_block_02, NUM_BLOCKS, 10);
 }
 
+static void test_mem_block_get(void)
+{
+	int i, ret, val;
+	void *block, *block1;
+
+#ifdef CONFIG_SYS_MEM_BLOCKS_LISTENER
+	listener_idx = 0;
+	heap_listener_register(&mem_block_01_alloc);
+	heap_listener_register(&mem_block_01_free);
+#endif
+
+	/* get a 2 entiries memory block starting from 0 */
+	ret = sys_mem_blocks_get(&mem_block_01, 0, 2, &block);
+	zassert_equal(ret, 0,
+		      "sys_mem_blocks_get failed (%d)", ret);
+
+	/* blocks 0 and 1 should be taken */
+	for (i = 0; i < NUM_BLOCKS; i++) {
+		ret = sys_bitarray_test_bit(mem_block_01.bitmap,
+					    i, &val);
```

Comment:
ret gets assigned but never checked; that's going to show up later as more work for some poor soul doing code coverage analysis.  Just a `zassert_equal(ret, 0, "API failure");` is enough.  This pattern repeats elsewhere too.

Reply:
fixed in all tests (not only added ones)